Sources/ApolloSQLite/SQLiteDotSwiftDatabase.swift

```diff
@@ -52,6 +52,13 @@ public final class SQLiteDotSwiftDatabase: SQLiteDatabase {
     let query = self.records.filter(keyColumn == cacheKey)
     try self.db.run(query.delete())
   }
+
+  public func deleteRecords(matching pattern: CacheKey) throws {
+    let wildcardPattern = "%\(pattern)%"
+    let query = self.records.filter(keyColumn.like(wildcardPattern))
```

Comment:
From what I could find it appears that [sqlite LIKE is case-insensitive by default](https://sqlite.org/lang_expr.html#the_like_glob_regexp_and_match_operators), hence nothing to explicitly state case insensitivity.

Reply:
There is now a test to validate this behaviour - https://github.com/apollographql/apollo-ios/pull/1848/files#diff-15a6945fd0d80fdabb91e30e1b1ffb06c396e8a7466602afb574f13b2c5b650fR613